ucs/database.cpp

```diff
@@ -268,9 +270,31 @@ bool Database::GetVariable(const char *varname, char *varvalue, uint16 varvalue_
 	return true;
 }
 
+
+void Database::LoadBlockChannels() {
+	LogInfo("Loading blocked channels from the database");
+
+	const std::string query = "SELECT `name`, `owner`, `password`, `minstatus` FROM `chatchannels` WHERE `owner` = '*Block*'";
```

Comment:
Having blocked channels being a magic string value in a column is not ideal, I'd rather us be explicit in a column on the behavior we desire from this channel.

A new column such as `reserved_name`

Reply:
Updated to use a separate table called `reserved_channel_names `.